mw-webapp/.storybook/preview.ts

```diff
@@ -1,4 +1,7 @@
 import type { Preview } from "@storybook/react";
+import { Theme, ThemeWorker } from './../src/utils/ThemeWorker';
+
+ThemeWorker.setTheme(Theme.LIGHT);
```

Comment:
Does it possible to add theme switcher somewhere in the ui, to get ability for user change components styles?

Reply:
Yes, it's possible, I can add ThemeSwitcher to storybook component like this, 
![image](https://github.com/tritonJS826/masters-way/assets/112805668/5516a04a-c4e5-4d5d-9ace-ebf0d892556d)
but we have problem with Defaut doc(autodocs) with switcher position .
![image](https://github.com/tritonJS826/masters-way/assets/112805668/b957cf32-53f7-4bae-bd97-d9add6f7c94b)

Reply:
we can turn off autodocs, the same options are available in the storybook component
![image](https://github.com/tritonJS826/masters-way/assets/112805668/9bf38fa4-a384-431d-ad71-d43a5194d52c)

Reply:
It's possible to add ThemeSwitcher to storybook component without docs page. Add this funcionality?
![image](https://github.com/tritonJS826/masters-way/assets/112805668/de97d83a-2976-463c-93cf-1fb87868fca9)
![image](https://github.com/tritonJS826/masters-way/assets/112805668/6857c2fd-10d1-4367-93dc-e908b3c039de)